src/index.html

```diff
@@ -3,11 +3,68 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link
+      rel="stylesheet"
+      href="./styles/index.scss"
+    >
+    <link rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
     <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card">
+      <img
```

Comment:
<img width="1312" alt="Screenshot 2023-06-29 at 16 01 45" src="https://github.com/mate-academy/layout_product-cards/assets/53228259/3e2acb0c-0b38-4330-a72b-e797d0f4f012">

Reply:
comment still not fixed
https://github.com/mate-academy/layout_product-cards/pull/3236/files#r1246590191